### ISSUE TITLE
Tahmina: Debugging

### ISSUE THREAD

**Issue by tahminarasoli** [HackYourFutureBelgium/class-11-12#162]
> [module repository](_) 

### What is JavaScript?
- [x] Andrew Mosh
- [x] Code School
- [x] javascript.info

### Debugging Tools
- [x] Errors & Debugging
- [x] Developer Console

### Getting Started
- [x] Install or update the study-lenses package globally 
- [x] Clone this repository 
- [x] cd into the repository 
- [x] Open the repository using study
- [x] The material will open in your default browser

---

## Week 1

---

- [X] Sunday Prep

- [X] Sunday Class

### Check-In

__Progress__
practice debugging 
- [X] Variables
- [X] Primitive Types
- [X] Comparing and Asserting
- [X] Explicit Coercion
- [X] logical-operators
- [X] Block Scope

__Blocked__
the difficult one was **Object.is** for me.
__Next Steps__

__Tip(s) of the week__

---

**Comment by colevandersWands** [comment]
Hello @tahminarasoli , How's it going with `Object.is`?

**Comment by tahminarasoli** [comment]
const _1_expect = _;  **// how the blank space is true here? does it compare the UNICODE values?**
const _1_actual = "1" >= 1;
console.assert(_1_actual === _1_expect, "Test 1");

**Comment by colevandersWands** [comment]
@tahminarasoli , can you share the rest of your file?  normally this should throw an error, you probably declared a variable called `_` higher up in your file

the comparison `"1" >= 1` is `true` because JavaScript will convert both values to a number, then compare the number values.

**Comment by tahminarasoli** [comment]
> @tahminarasoli , can you share the rest of your file? normally this should throw an error, you probably declared a variable called `_` higher up in your file
> 
> the comparison `"1" >= 1` is `true` because JavaScript will convert both values to a number, then compare the number values.

yes, it is [here](http://localhost:4600/1-isolate/09-comparisons/exercises/greater-than-or-equal-to.js?study) .

**Comment by colevandersWands** [comment]
Hey @tahminarasoli , sorry for the late reply. I had first thought you meant the program wasn't throwing an error on `_1_expect`.  Now I see you were asking why the correct answer is `true`?

It's not comparing unicode values, it's comparing them both as numbers.  `Number("1") === 1` and 1 is greater than or equal to itself

(
